test/CMakeLists.txt

```diff
@@ -52,3 +52,22 @@ target_link_libraries(checkRandomStateTest ${Boost_UNIT_TEST_FRAMEWORK_LIBRARIES
 target_link_libraries(checkRandomStateTest ethereum)
 target_link_libraries(checkRandomStateTest ethcore)
 
+enable_testing()
+set(CTEST_OUTPUT_ON_FAILURE TRUE)
+
+include(EthUtils)
+
+eth_add_test(InterfaceStub
+	ARGS --eth_testfile=BlockTests/bcJS_API_Test --eth_threads=1
+	ARGS --eth_testfile=BlockTests/bcJS_API_Test --eth_threads=3
+	ARGS --eth_testfile=BlockTests/bcJS_API_Test --eth_threads=10
+	ARGS --eth_testfile=BlockTests/bcValidBlockTest --eth_threads=1
+	ARGS --eth_testfile=BlockTests/bcValidBlockTest --eth_threads=3
+	ARGS --eth_testfile=BlockTests/bcValidBlockTest --eth_threads=10
+)
```

Comment:
@CJentzsch 
This might interest you. First param is the name of boost test suite. Then I define with what additional arguments it should be called with. Above code is similar to:
```bash
testeth --run_test=InterfaceStub --eth_testfile=BlockTests/bcJS_API_Test --eth_threads=1
testeth --run_test=InterfaceStub --eth_testfile=BlockTests/bcJS_API_Test --eth_threads=3
testeth --run_test=InterfaceStub --eth_testfile=BlockTests/bcJS_API_Test --eth_threads=10
testeth --run_test=InterfaceStub --eth_testfile=BlockTests/bcValidBlockTest --eth_threads=1
testeth --run_test=InterfaceStub --eth_testfile=BlockTests/bcValidBlockTest --eth_threads=3
testeth --run_test=InterfaceStub --eth_testfile=BlockTests/bcValidBlockTest --eth_threads=10
```

But in addition, it is run in parallel using ctest. It also adds `test.InterfaceStub` to target list, so it is possible to trigger the test from command line by typing:

```bash
make test.InterfaceStub
```